tests/framework/installer/ceph_manifests.go

```diff
@@ -355,6 +357,8 @@ spec:
                 properties:
                   failureDomain:
                     type: string
+                  crushRoot:
+				  	type: string
```

Comment:
The spacing looks off

Reply:
i am not sure but why i am able to see this on my local machine editor i.e. goland.